parsers/nmparser/parse_lst_file_test.go

```diff
@@ -226,12 +227,38 @@ func TestSetLargeConditionNumber(t *testing.T) {
 		// 	largeConditionNumber: Undefined,
 		// 	context:              "not set",
 		// },
+
+		{
+			lines: []string{
```

Comment:
More of a comment than anything, you can store multiline text components with the backtick operator: 

```go
things := `
I'm a multi
line string
`
```